src/DynamoCoreWpf/Controls/StartPage.xaml.cs

```diff
@@ -417,6 +432,13 @@ private void OnStartPageLoaded(object sender, RoutedEventArgs e)
 
             var id = Wpf.Interfaces.ResourceNames.StartPage.Image;
             StartPageLogo.Source = dynamoViewModel.BrandingResourceProvider.GetImageSource(id);
+
+            if (dynamoViewModel.Model.PreferenceSettings.ShowGaleryAtStartUp)
+            {
+                dynamoViewModel.Model.PreferenceSettings.ShowGaleryAtStartUp = false;
+                dynamoViewModel.ShowGalleryCommand.Execute(null);
```

Comment:
It seems like then we cannot have `ShowGalleryAtStartUp` always set to `true` anymore, perhaps the condition should have been:

```csharp
if (PreferenceSettings.IsFirstRun || PreferenceSettings.ShowGalleryAtStartUp)
{
    dynamoViewModel.ShowGalleryCommand.Execute(null);
}
```

Of course that way we need a check box on the start page that says *Always show at start-up*, not sure if @elayabharath will buy into that idea or not :)